src/iosrv/init.c

```diff
@@ -63,7 +63,7 @@ static char	       *daos_sysname = DAOS_DEFAULT_SYS_NAME;
 const char	       *dss_storage_path = "/mnt/daos";
 
 /** NVMe config file */
-const char	       *dss_nvme_conf = "/etc/daos_nvme.conf";
+const char	       *dss_nvme_conf = "";
```

Comment:
Would this work better?
```suggestion
const char	       *dss_nvme_conf;
```